src/bitcoin-cli.cpp

```diff
@@ -373,16 +368,9 @@ class NetinfoRequestHandler : public BaseRequestHandler
 {
 private:
     static constexpr uint8_t MAX_DETAIL_LEVEL{4};
-    std::array<std::array<uint16_t, NETWORKS.size() + 1>, 3> m_counts{{{}}}; //!< Peer counts by (in/out/total, networks/total)
+    std::map<std::string, std::array<int, 3>> m_counts; //!< Peer counts by network type (in / out / total)
```

Comment:
Total is just in+out, and turning the std::array into a std::pair gets default zero initialisation, which would be useful later. We only need total one place, so IMO we should just sum it there rather than counting one by one.

Reply:
fixed in bf69a13677680dbf2065e18e3e5c7882ec3aab33 -- the only messy part was that `std::pair` values can't be retrieved by variable like `i` in the `for` loop, so I put a switch in there so that 0 = first and 1 = second, I dunno if that's best.